app/views/users/require_2sv.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :title, "Create new user" %>
+
+<h1>2-step verification settings for new user</h1>
+
+<%= form_for @user, :html => {:class => 'well'} do |f| %>
+  <%= f.hidden_field :skip_update_user_permissions, value: "true" %>
+  <p class="checkbox">
+    <%= f.label :require_2sv do %>
+      <%= f.check_box :require_2sv %> Mandate 2-step verification for this user <%= "(this will remove their exemption)" if @user.exempt_from_2sv? %>
+    <% end %>
+    <br/>
+    User will be prompted to set up 2-step verification again the next time they sign in.
+  </p>
+
+  <%= f.submit "Update user", :class => 'btn btn-success' %>
+<% end %>
```

Comment:
Nitpick: newline missing

Reply:
Good spot.  I've just moved from Atom to VSCode and hadn't enabled the setting to automatically add new-lines at end of file yet.